spec/models/user_spec.rb

```diff
@@ -118,39 +98,17 @@
         user.reload
         user.mobile = num
         user.save
-        expect(user.errors.any?).to be_falsey
+        expect(user.errors.any?).to be_falsey, user.errors.messages.inspect
```

Comment:
There are a bunch of be_falsey and be_truthy in this file.  I suggest we made that as a separate change.   Or did you mean my addition of the inspect?